erpnext/hr/doctype/shift_assignment/shift_assignment.py

```diff
@@ -16,7 +16,7 @@ def validate(self):
 		self.validate_overlapping_dates()
 
 		if self.end_date and self.end_date <= self.start_date:
-			frappe.throw(_("End Date should not be less than Start Date"))
+			frappe.throw(_("End Date must not be greater than Start Date"))
```

Comment:
```suggestion
			frappe.throw(_("End Date must not be lesser than Start Date"))
```
My mistake. apologies